### DISCUSSION TITLE
角色控制器组件 demo 代码本地运行报错

### DISCUSSION THREAD

**Discussion by liangxiaowei** [galacean/discussion#25] [Engine] [answered]
引擎版本：0.8.0-beta.30
toolkit版本：0.8.0-beta.3
physics-physx版本：0.8.0-beta.30

问题描述及复现：
![image](https://user-images.githubusercontent.com/8638403/195489955-01c979ce-9bb3-4eb0-a21f-1ea067a1bda2.png)

想在本地运行角色控制组件的demo，但是直接复制官方demo代码运行会报错，经过注释调试，发现是 animator 播放动画导致的。把  下图 animator.play(State.Idle); 注释掉就不会报错，并且纹理可以正常加载

![image](https://user-images.githubusercontent.com/8638403/195492551-d736c30e-ac22-43e5-b317-17329d1a5137.png)

[oasisDemo.zip](https://github.com/oasis-engine/engine/files/9770038/oasisDemo.zip)

**Comment by luzhuang** [comment]
[oasisDemo 2.zip](https://github.com/oasis-engine/engine/files/9770210/oasisDemo.2.zip)
我这里直接使用官方demo是ok的啊

**Reply by liangxiaowei** [reply]
我断点调试了一下，动画数据这里的 _valueType 是 undefined
![image](https://user-images.githubusercontent.com/8638403/195518549-b2b2c45f-5736-4d55-a1c2-aa222acd2791.png)

**Reply by liangxiaowei** [reply]
解析 gltf 后，这里没有一个分支命中，key.value 的 constructor 在浏览器显示是 "Vector35"，不知道为什么 instanceof Vector3 是 false，我自己重新new Vector3，instanceof Vector3 是 true 的
![image](https://user-images.githubusercontent.com/8638403/195524738-e7625a6a-fc57-4ef0-959d-9ebfbd7e7932.png)

**Comment by liangxiaowei** [comment, marked as answer]
我把 node_module 删除了，重新 install 就好了，估计是有些缓存问题
